hazelcast-tpc-engine/src/test/java/com/hazelcast/internal/tpcengine/util/TpcTestSupport.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2008-2023, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.internal.tpcengine.util;
+
+import static org.junit.Assume.assumeFalse;
+
+public class TpcTestSupport {
+
+    private TpcTestSupport() {
+    }
+
+    public static void assumeThatNoWindowsOS() {
```

Comment:
There is already `TpcTestSupport` class can you move this method to there? Also I would prefer `assumeNotWindows` as it's more concise.

Reply:
Rev : e70a9997023efc566d039be295309cfd5c9d49d6